CMSgov/mint-app
# [EASI-2588] 6-pager Meeting Help Article

# EASI-2588

## Changes and Description

- Added the usual files added when creating a new page
- Added new props to `SolutionDetails/Modal` to allow for LDG resource panel to pop up and close in this help article




## How to test this change

1. Navigate to `http://localhost:3005/help-and-knowledge` and see that the 6-page card is there and click through
2. Ensure page matches figma
3. Click on links to make sure it goes the right places
4. Ensure the `LDG` link opens the resource panel and user is able to close it




## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
